src/hugolib/app_links_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Gotham Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package hugolib
+
+import (
+	"testing"
+
+	"github.com/gothamhq/gotham/deps"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func TestAppLinksOutput(t *testing.T) {
+
+	t.Parallel()
+
+	testCases := []struct {
+		prefix  string
+		bundle  string
+		passing bool
+	}{
+		{
+			"ABCDE12345",
+			"com.gothamhq.ios",
+			true,
+		},
+		{
+			"",
+			"com.gothamhq.ios",
+			false,
+		},
+		{
+			"",
+			"ABCDE12345",
```

Comment:
If you're trying to follow the setup I did with Digital Asset Links, this is flipped. Line 34 and 35 should be swapped.

Reply:
Good catch!